static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -326,8 +335,15 @@ class RuleConditionsForm extends PureComponent<Props, State> {
   }
 
   renderInterval() {
-    const {organization, disabled, alertType, timeWindow, onTimeWindowChange, project} =
-      this.props;
+    const {
+      organization,
+      disabled,
+      alertType,
+      timeWindow,
+      onTimeWindowChange,
+      project,
+      monitorType,
+    } = this.props;
```

Comment:
we're using a lot of props here - i wonder if we could use hooks or reduce the prop drilling 🤔